config/locales/registrations.en.yml

```diff
@@ -27,6 +27,11 @@ en:
           convictions_check_button: "Check conviction matches"
           revert_to_payment_summary_button: "Send back to payment summary"
           missed_worldpay_payment_button: "Add a missed WorldPay payment"
+      tier:
```

Comment:
This is in config/locales/shared/registrations.en.yml as well now – should this have been added here? (Same goes for renewing_registrations.en.yml)

Reply:
Nope :/ conflicts madness XD